File: LeetCode/0717-1-bit-and-2-bit-characters/solution.py
class Solution01(object):
    """线性扫描"""

    def isOneBitCharacter(self, bits):
        """
        :type bits: List[int]
        :rtype: bool
        """
        i = 0
        while i < len(bits) - 1:
            i += bits[i] + 1
        return i == len(bits) - 1

File: LeetCode/0717-1-bit-and-2-bit-characters/Solution.java
public class Solution {}

// 线性扫描
class Solution01 {
	public boolean isOneBitCharacter(int[] bits) {
		int i = 0;
		while (i < bits.length - 1) {
			i += bits[i] + 1;
		}
		return i == bits.length - 1;
	}
}